specs/saltpack_encryption_v1.md

```diff
@@ -95,8 +95,12 @@ The header packet is a MessagePack list with these contents:
 - The **sender secretbox** is a
   [`crypto_secretbox`](http://nacl.cr.yp.to/secretbox.html) containing the
   sender's long-term public key, encrypted with the **payload key** from below.
-- The **recipients list** contains a recipient pair for each recipient key,
-  including an encrypted copy of the **payload key**. See below.
+- The **recipients list** contains a recipient pair for each recipient
+  key, including an encrypted copy of the **payload key** (see
+  below). Note that a MessagePack array can hold
+  [at most 2³² &minus; 1](https://github.com/msgpack/msgpack/blob/master/spec.md#array-format-family)
```

Comment:
Unicode shenanigans :-D

Were you able to test rendering this to make sure it looks ok?

Reply:
Yeah, I did so by modifying the HTML on the saltpack webpage with Chrome inspector.